src/renderers/shared/reconciler/__tests__/ReactComponent-test.js

```diff
@@ -38,7 +38,7 @@ describe('ReactComponent', function() {
     );
 
     expect(function() {
-      React.render(<div></div>, null);
+      React.render(<div></div>, {});
```

Comment:
For the `render()` warning. According to the [source code](https://github.com/facebook/react/blob/master/src/renderers/dom/client/ReactMount.js#L521),
```
warning(
      container && container.tagName !== 'BODY',
      'render(): Rendering components directly into document.body is ' +
      'discouraged, since its children are often manipulated by third-party ' +
      'scripts and browser extensions. This may lead to subtle ' +
      'reconciliation issues. Try rendering into a container element created ' +
      'for your app.'
    );
```
When the `container` is false, the warning message would be confused .